Emitron/Emitron/UI/SceneDelegate.swift

```diff
@@ -41,29 +41,13 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     // This delegate does not imply the connecting scene or session are new (see `application:configurationForConnectingSceneSession` instead).
 
     // TODO: When a modifier is available these should be refactored
```

Comment:
If we're gutting all these lines we should remove the TODO as well

Reply:
Currently researching a modification that allows modifying navbar transparency on scroll.  Not sure if possible but will find out.  Will amend this comment based on what i learn.